scripts/rabbitmq-server-ha.ocf

```diff
@@ -260,6 +262,16 @@ ${OCF_RESKEY_binary} should listen on this port
 <content type="boolean" default="${OCF_RESKEY_node_port_default}" />
 </parameter>
 
+<parameter name="vhost" unique="0" required="0">
+<longdesc lang="en">
+Main virtual host used in the cluster, guaranteed to not suffer data loss. Other
```

Comment:
This is a bold claim. Under what conditions? What or who offers the guarantee?

There is no "main" or "special" vhost in RabbitMQ. Please reword this to be less confusing.